package.json

```diff
@@ -20,106 +20,26 @@
-    "babel-cli": "^6.10.1",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.3",
-    "babel-jest": "^23.6.0",
-    "babel-loader": "^7.1.1",
-    "babel-plugin-dynamic-import-node": "^2.0.0",
-    "babel-plugin-lodash": "^3.3.4",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-assign": "^6.8.0",
-    "babel-plugin-transform-object-rest-spread": "^6.8.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.5.0",
-    "coveralls": "^3.0.0",
-    "enzyme": "^3.4.0",
-    "enzyme-adapter-react-16": "^1.4.0",
-    "enzyme-to-json": "^3.2.1",
-    "eslint": "^4.10.0",
-    "eslint-import-resolver-babel-module": "^4.0.0",
-    "eslint-plugin-patternfly-react": "0.2.0",
-    "identity-obj-proxy": "^3.0.0",
-    "jed": "^1.1.1",
-    "jest-cli": "^23.6.0",
-    "jest-prop-type-error": "^1.1.0",
-    "node-sass": "^4.5.0",
-    "patternfly": "^3.58.0",
+    "@babel/core": "^7.7.0",
+    "@theforeman/builder": "^4.0.2",
+    "@theforeman/stories": "^4.0.2",
+    "@theforeman/test": "^4.0.2",
+    "@theforeman/vendor-dev": "^4.0.2",
+    "@theforeman/eslint-plugin-foreman": "^4.0.2",
+    "babel-eslint": "^10.0.0",
+    "eslint": "^6.7.2",
     "prettier": "^1.13.5",
     "raf": "^3.4.0",
-    "react-redux-test-utils": "^0.1.1",
-    "react-remarkable": "^1.1.3",
-    "sass-loader": "^6.0.7",
     "stylelint": "^9.3.0",
     "stylelint-config-standard": "^18.0.0",
     "surge": "^0.20.3"
   },
   "dependencies": {
-    "babel-polyfill": "^6.26.0",
-    "classnames": "^2.2.5",
-    "lodash": "^4.17.10",
-    "patternfly-react": "^2.19.1",
-    "prop-types": "^15.6.0",
-    "react": "^16.4.0",
-    "react-dom": "^16.4.0",
-    "react-intl": "^2.8.0",
-    "react-redux": "^5.0.6",
-    "redux": "^3.6.0",
-    "redux-thunk": "^2.2.0",
-    "reselect": "^3.0.1",
-    "seamless-immutable": "^7.1.2",
-    "urijs": "^1.18.10",
-    "uuid": "^3.0.1"
-  },
-  "jest": {
-    "automock": true,
-    "verbose": true,
-    "testMatch": [
-      "**/*.test.js"
-    ],
-    "testURL": "http://localhost/",
-    "collectCoverage": true,
-    "collectCoverageFrom": [
-      "webpack/**/*.js",
-      "!webpack/index.js",
-      "!webpack/test_setup.js",
-      "!webpack/**/bundle*",
-      "!webpack/stories/**",
-      "!webpack/**/*stories.js"
-    ],
-    "coverageReporters": [
-      "lcov"
-    ],
-    "unmockedModulePathPatterns": [
-      "webpack/",
-      "react",
-      "node_modules/"
-    ],
-    "moduleNameMapper": {
-      "^.+\\.(png|gif|css|scss)$": "identity-obj-proxy"
-    },
-    "globals": {
-      "__testing__": true
-    },
-    "transform": {
-      "^.+\\.js$": "babel-jest"
-    },
-    "moduleDirectories": [
-      "node_modules",
-      "webpack"
-    ],
-    "setupFiles": [
-      "raf/polyfill",
-      "jest-prop-type-error",
-      "./webpack/test_setup.js"
-    ]
+    "jed": "^1.1.1",
+    "react-intl": "^2.8.0"
```

Comment:
Do you actually use `jed` and `reat-intl`?

Reply:
yes, using `react-intl` for relative time
and `jed` for sprintf